safety-shim.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
```

Comment:
Could we `echo` something here stating that safety is unsupported, and we're running pip-audit instead?

Reply:
all right